modin/pandas/series.py

```diff
@@ -1246,9 +1246,9 @@ def nlargest(self, n=5, keep="first"):  # noqa: PR01, RT01, D200
             raise NotImplementedError(
                 "Series.nlargest is not implemented for empty Series."
             )
-        return DataFrame(
```

Comment:
hmmm why were we returning a Series before?

Reply:
Returning a DataFrame was introduced in this PR (https://github.com/devin-petersohn/modin/pull/39/files), but that was a bug.